modules/nf-core/vgan/haplocart/main.nf

```diff
@@ -0,0 +1,48 @@
+process VGAN_HAPLOCART {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::vgan=1.0.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vgan:1.0.1--h9ee0642_0':
+        'quay.io/biocontainers/vgan:1.0.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    path("hc_files")
```

Comment:
```suggestion
    path(hc_files)
```

Reply:
Unlss you want to name it specifically as hc_files, in which case `path(hc_files, stageAs: 'hc_files/')` [^1]

[^1] https://www.nextflow.io/docs/latest/process.html?highlight=stageas#input-type-path ( see bottom of there)